### DISCUSSION TITLE
Support Password Managers from MAUI Entry fields

### DISCUSSION THREAD

**Discussion by mobilewares** [dotnet/maui#18966] [Q&A] [answered]
Hi -

We have a login page in our MAUI App with a UserName and Password MAUI Entry fields. When it's running on iOS (and Android) the OS is not providing any visual hooks into Password Managers (such as Lastpass. Keystore etc) on the popup keyboard as it normally would for other Apps/Browser. Right now the only way to get login credentials entered easily (without typing) is to launch Lastpass or other Password manager and switch back/forth using copy/paste - which is not ideal.

The Xamarin version of our App previously worked fine for this & both fields could be auto filled.

Just wondering if anyone knows how to enable this functionality for MAUI. Is there some special tricks or Automation ID's that need to be used?  Note that we don't currently need any advanced AutoFill functionality (ie. auto pasting of Security Code from received SMS)  - just basic support for password managers.

Thanks in advance for any help/tips.

Niall

**Comment by mobilewares** [comment, marked as answer]
nvm - have solved this one..

Here's the fixes I did if anyone interested : 
- Login Entry (Email) - set properties **Keyboard**="Email" and **AutomationId**="Login"
- Password Entry - set properties **Keyboard**="Default" and **AutomationId**="Password" (and IsPassword="True" of course)

**Reply by Kapusch** [reply]
Note: you shouldn't need to define AutomationId to achieve this..
